pepys_import/file/e_trac_importer.py

```diff
@@ -134,6 +134,9 @@ def _load_this_file(self, data_store, path, file_object, datafile):
             state.speed = speed
             speed_token.record(self.name, "speed", speed, "knots")
 
+            state.platform_name = vessel_name
```

Comment:
I'm nervous about  the way we're storing this cached data in the state, since it relies on the parser author remembering to use the lines.  Can't we retrieve/store these values inside the `datafile.createState()` method?  Then it's under our control, and we know it will be done every time.

Reply:
We're also relying on the parser author storing the _correct_ values in the state.  Again, if it happens in our code we can rely on it.

Reply:
Yes, we can do that. I'm changing the code.